### ISSUE TITLE
Missing tokens 

### ISSUE THREAD

**Issue by minhchu** [cschroeter/park-ui#237]
I've just did a fresh installation of latest NextJs, PandaCss and ParkUI.

But there are some warnings from PandaCss log: 

```
🐼 warn [config] ⚠️ Invalid config:
- [tokens] Missing token: `colors.white` used in `config.semanticTokens.colors.neutral.default`
- [tokens] Missing token: `colors.black` used in `config.semanticTokens.colors.neutral.default`
- [tokens] Missing token: `colors.black` used in `config.semanticTokens.colors.neutral.fg`
- [tokens] Missing token: `colors.white` used in `config.semanticTokens.colors.neutral.fg`
- [tokens] Missing token: `colors.white` used in `config.semanticTokens.colors.neutral.text`
- [tokens] Missing token: `colors.black` used in `config.semanticTokens.colors.neutral.text`
- [tokens] Missing token: `colors.white` used in `config.semanticTokens.colors.gray.default`
- [tokens] Missing token: `colors.black` used in `config.semanticTokens.colors.gray.default`
- [tokens] Missing token: `colors.black` used in `config.semanticTokens.colors.gray.fg`
- [tokens] Missing token: `colors.white` used in `config.semanticTokens.colors.gray.fg`
- [tokens] Missing token: `colors.white` used in `config.semanticTokens.colors.gray.text`
- [tokens] Missing token: `colors.black` used in `config.semanticTokens.colors.gray.text`
- [tokens] Missing token: `colors.white` used in `config.semanticTokens.colors.accent.default`
- [tokens] Missing token: `colors.black` used in `config.semanticTokens.colors.accent.default`
- [tokens] Missing token: `colors.black` used in `config.semanticTokens.colors.accent.fg`
- [tokens] Missing token: `colors.white` used in `config.semanticTokens.colors.accent.fg`
- [tokens] Missing token: `colors.white` used in `config.semanticTokens.colors.accent.text`
- [tokens] Missing token: `colors.black` used in `config.semanticTokens.colors.accent.text`
- [tokens] Missing token: `colors.white` used in `config.semanticTokens.colors.bg.default`
```
My `package.json`: 

```
{
  "name": "backend-nestjs-course",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "prepare": "panda codegen",
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint"
  },
  "dependencies": {
    "@ark-ui/react": "^2.0.2",
    "next": "14.1.0",
    "react": "^18.2.0",
    "react-dom": "^18.2.0"
  },
  "devDependencies": {
    "@pandacss/dev": "^0.31.0",
    "@park-ui/panda-preset": "^0.34.1",
    "@types/node": "^20",
    "@types/react": "^18.2.55",
    "@types/react-dom": "^18.2.19",
    "eslint": "^8",
    "eslint-config-next": "14.1.0",
    "typescript": "^5"
  }
}
```
My `panda.config.ts`: 

```
import { defineConfig } from "@pandacss/dev";

export default defineConfig({
  // Whether to use css reset
  preflight: true,

  jsxFramework: "react",

  presets: ["@pandacss/preset-base", "@park-ui/panda-preset"],

  // Where to look for your css declarations
  include: ["./src/**/*.{js,jsx,ts,tsx}", "./pages/**/*.{js,jsx,ts,tsx}"],

  // Files to exclude
  exclude: [],

  // Useful for theme customization
  theme: {
    extend: {},
  },

  // The output directory for your css system
  outdir: "styled-system",
});
```

**Comment by cschroeter** [comment]
@minhchu 

Thanks for taking your time to create this issue. I think this is a bug in Panda itself since the color tokens are declared correctly:

```jsx
defineTokens.colors({
  black: {
    DEFAULT: { value: '#000000' },
    a1: { value: 'rgba(0, 0, 0, 0.05)' },
    // ..
    a12: { value: 'rgba(0, 0, 0, 0.95)' },
  },
  white: {
    DEFAULT: { value: '#ffffff' },
    a1: { value: 'rgba(255, 255, 255, 0.05)' },
    // ..
    a12: { value: 'rgba(255, 255, 255, 0.95)' },
  },
})
```

The reason for this is most likely the special `DEFAULT` syntax. I think we can rid off this warning by fallback to just black and white from the browser.

**Comment by cschroeter** [comment]
Fixed in `0.35.1`

**Comment by minhchu** [comment]
Wow thank you, that was fast :zap:
